Add Environment Variables Section (#4)
agamm/awesome-developer-first: README.md

* [Databases & Spreadsheets](#databases--spreadsheets)
* [Deployment Hosting](#deployment-hosting)
* [Discussions](#discussions)

> * [Environment Variables](#environment-variables)

Reviewer: ```suggestion
* [Environment & Secret Management](#environment--secret-management)
```

* [Yotpo](https://www.yotpo.com/platform/reviews/) - Reviews ratings and Q&A as a service.
* [Gitter](https://gitter.im) - Chat and networking platform for developers.

> ## Environment Variables

Reviewer: ```suggestion
## Environment & Secret Management
```

> *Manage environment variables.*

Reviewer: ```suggestion
*Manage environment variables and secrets for multiple apps or projects.*
* [Vault](https://www.vaultproject.io/) -  API driven secret key management.
```

> * [Doppler](https://doppler.com/) - All your environment variables across projects and environments, in one place.

Reviewer: ```suggestion
* [Doppler](https://doppler.com/) - Environment variable management across projects, in one place.
* [1Password  Secrets Automation](https://1password.com/secrets/) - Secrets management with integration to 1Password.
```

> 

Reviewer: ```suggestion
* [Envkey](https://www.envkey.com/) - Environment management with syncing mechanism for all apps.
```

* [Yotpo](https://www.yotpo.com/platform/reviews/) - Reviews ratings and Q&A as a service.
* [Gitter](https://gitter.im) - Chat and networking platform for developers.

## Environment & Secret Management

*Manage environment variables and secrets for multiple apps or projects.*

> * [Vault](https://www.vaultproject.io/) -  API driven secret key management.

Reviewer: ```suggestion
* [Vault](https://www.vaultproject.io/) - API driven secret key management.
```

* [Doppler](https://doppler.com/) - Environment variable management across projects, in one place.

> * [1Password  Secrets Automation](https://1password.com/secrets/) - Secrets management with integration to 1Password.

Reviewer: ```suggestion
* [1Password Secrets Automation](https://1password.com/secrets/) - Secrets management with integration to 1Password.
```
